frontend/rust-lib/flowy-database2/src/services/field/type_options/type_option_cell.rs

```diff
@@ -100,7 +98,6 @@ impl AsRef<u64> for CellDataCacheKey {
 struct TypeOptionCellDataHandlerImpl<T> {
   inner: T,
   cell_data_cache: Option<CellCache>,
-  cell_filter_cache: Option<CellFilterCache>,
```

Comment:
what is the benefit that removes the cache?

Reply:
I haven't tested the performance, so I can't say whether this is much faster. I remove it for functional reasons. The previous filtering logic assumes one filter per field, and so the `CellFilterCache` is keyed by the `field_id`. With advanced filters, multiple filters can be applied to a single field.

Reply:
Got it